blog/2018-06-21-debezium-0-8-0-beta1-released.adoc

```diff
@@ -0,0 +1,148 @@
+we decided to alter our versioning schema a little bit:
+going forward we may do one more Beta and CR ("candidate release") releases before doing a final one.
+This will allow us to get feedback from the community early on,
+while still completing and polishing specific features.
+Final (stable) releases will be named like 0.8.0.Final etc.
+
+This release would not have been possible without our fine community;
+a huge "thank you" goes out to the following open source enthusiasts who all contributed to the new version:
+https://github.com/echo-xu[Echo Xu],
+https://github.com/vuckooo[Ivan Vucina],
+https://github.com/glistman[Listman Gamboa],
+https://github.com/omarsmak[Omar Al-Safi],
+https://github.com/pgoranss[Peter Goransson],
+https://github.com/kucharo2[Roman Kuchar] (who did a tremendous job with the new DDL parser implementation!),
+https://github.com/sagarrao[Sagar Rao],
+https://github.com/sauliusvl[Saulius Valatka],
+https://github.com/sairam881990[Sairam Polavarapu],
+https://github.com/Crim[Stephen Powis] and
+https://github.com/sweat123[WenZe Hu].
+
+Thank you all for your help!
+
+Now let's take a closer look at some of the features new in Debezium 0.8.0.Beta1;
+as always, you can find the complete list of changes of this release in the link:/docs/releases/#release-0-8-0-beta-1[change log].
+Take a special look at the breaking changes and the upgrade notes.
+
+== XStream-based Oracle Connector (Preview)
+
+Support for a Debezium Oracle connector has been one of the most asked for features for a long time
+(its original issue number is https://issues.jboss.org/browse/DBZ-20[DBZ-20]!).
+So we are very happy that we eventually can release a first work-in-progress version of that connector.
+At this point this code is still very much evolving, so it should be considered as a first tech preview.
+This means it's not feature complete (most notably, there's no support for initial snapshots yet),
+the emitted message format may still change etc.
+So while we don't recommend using it in production quite yet,
+you should definitely give it a try and report back about your experiences.
+
+One challenge for the Oracle connector is how to get the actual change events out of the database.
+Unlike with MySQL and Postgres, there's unfortunately no free-to-use and easy-to-work-with API which would allow to do the same for Oracle.
+After some exploration we decided to base this first version of the connector on the https://docs.oracle.com/database/121/XSTRM/xstrm_intro.htm#XSTRM72647[Oracle XStream] API.
+While this (kinda) checks the box for "easy-to-work-with", it doesn't do so for "free-to-use":
+using this API requires you to have a license for Oracle's separate GoldenGate product.
+We're fully aware of this being not ideal, but we decided to still go this route as a first step,
+allowing us to get some experiences with Oracle and also get a connector into the hands who happen to have the required license already.
```

Comment:
IIRC the license is not need for devolepr purposes so it might be too strict statement.